packages/dev/core/src/PostProcesses/RenderPipeline/Pipelines/defaultRenderingPipeline.ts

```diff
@@ -787,6 +804,14 @@ export class DefaultRenderingPipeline extends PostProcessRenderPipeline implemen
             this._scene.getEngine().onResizeObservable.remove(this._resizeObserver);
             this._resizeObserver = null;
         }
+        if (this._activeCameraChangedObserver) {
+            this._scene.onActiveCameraChanged.remove(this._activeCameraChangedObserver);
+            this._activeCameraChangedObserver = null;
+        }
+        if (this._activeCamerasChangedObserver) {
+            this._scene.onActiveCamerasChanged.remove(this._activeCamerasChangedObserver);
+            this._activeCamerasChangedObserver = null;
+        }

```

Comment:
it's enough to remove the observer, even if it is null. the `remove` function takes null as well.